src/table/body-cell/styles.scss

```diff
@@ -32,6 +33,21 @@ $success-icon-padding-right: calc(#{$edit-button-padding-right} + #{$icon-width-
   word-wrap: break-word;
   border-bottom: awsui.$border-divider-list-width solid awsui.$color-border-divider-secondary;
   font-weight: inherit;
+
+  @include focus-visible.when-visible {
+    @include styles.focus-highlight(
+      (
+        'vertical': calc(-1 * #{awsui.$space-scaled-xxs}),
+        'horizontal': calc(-1 * #{awsui.$space-scaled-xxs}),
+      )
+    );
+    // @mixin focus-highlight sets cell's position to "relative".
```

Comment:
WDYT about adding an optional parameter to the mixin instead to override the `position`, to avoid these CSS overrides?

Reply:
That is technically possible but as there is no named arguments I'd have to pass the defaults for border-radius and box-shadow plus include it twice (for normal and for sticky cell). I think that is increasing the complexity.